.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4']
```

Comment:
IIRC shivammathur/setup-php supports 5.3+ directly so this could ease the maintenance.

Reply:
https://github.com/shivammathur/setup-php?tab=readme-ov-file#tada-php-support seems to agree, indeed